cardano-cli/src/Cardano/CLI/Byron/Parsers.hs

```diff
@@ -21,7 +21,7 @@ module Cardano.CLI.Byron.Parsers
   , parseUpdateVoteThd
   ) where
 
-import           Cardano.Prelude hiding (option)
+import           Cardano.Prelude
```

Comment:
This change seems to force the `option` -> `Opt.option` changes below?

Reply:
For some reason I had to do this for `ghc-8.10.7`.